stdlib/ballerina-builtin/src/main/ballerina/ballerina/collections/vector.bal

```diff
@@ -0,0 +1,120 @@
+package ballerina.collections;
```

Comment:
Don't we need functions like **set/replace** and **clear** (clear the vector) ?

Reply:
+1 will add those